src/Screens/Main/BuddyList/index.tsx

```diff
@@ -45,12 +45,11 @@ export default ({ navigation }: Props) => {
     navigation.navigate('Main/Chat', { buddyId });
   };
 
-
   const navigateToBanned = () => {
     navigation.navigate('Main/BuddyList/BannedList', {});
   };
 
-  const dropdown: Item[] = [
+  const items: Item[] = [
     { text: 'main.chat.navigation.banned', action: navigateToBanned, id: '1' },
```

Comment:
Not related to this PR but action as name is not good IMO. onPress would be better. This 'action' thing is a function that is ran on press.